fusor-ember-cli/app/templates/cloudforms/cfme-configuration.hbs

```diff
@@ -5,14 +5,14 @@
                cssId="cfme_root_password"
                isRequired=true
                disabled=isStarted
-               minChars=8
+               validator=passwordValidator
```

Comment:
@cfchase, what about leaving in the `minChars` as a variable? Not sure all password validations will be minimum of 8 chars